test/integration/targets/iam_user_info/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+test_group: '{{ resource_prefix }}-group'
+test_path: '/'
```

Comment:
If you're testing searching with paths it's worth testing with something like `/{{ resource_prefix }}/` too

Reply:
Only the default path ('/') is possible here since [iam_user](https://docs.ansible.com/ansible/latest/modules/iam_user_module.html) doesn't support the [`Path`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.Client.create_user) parameter.